packages/icons/src/components/CryptoIconWithPercentage.tsx

```diff
@@ -40,8 +38,8 @@ export const CryptoIconWithPercentage = ({
     const path = Skia.Path.Make();
     path.addCircle(CANVAS_SIZE / 2, CANVAS_SIZE / 2, RADIUS);
 
-    const percentageFill = useValue(0);
     const animationProgress = useSharedValue(0);
+    const percentageFill = useDerivedValue(() => mix(animationProgress.value, 0, 0.999));
```

Comment:
Can you add this comment? 
      // It's neccessary to clamp the value to 0.999 because Skia doesn't support values greater than 1 as end value.
        // And due to javascript floating point precision, 1 is not exactly 1 but little bit more sometimes.

Reply:
good catch, done